checkov/terraform/graph_builder/variable_rendering/renderer.py

```diff
@@ -366,6 +379,16 @@ def evaluate_non_rendered_values(self) -> None:
                     changed_attributes[attribute] = evaluated
             self.local_graph.update_vertex_config(vertex, changed_attributes)
 
+    @staticmethod
+    def extract_dynamic_value_in_map(dynamic_value: str) -> str:
+        left_bracket_with_quotation = '["'
+        right_bracket_with_quotation = '"]'
+        left_bracket = '['
```

Comment:
can you move to global scope and uppercase the name to make them "proper" constants 😄